lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -177,9 +177,13 @@ class BlockedToMFMA : public mlir::RewritePattern {
       if (elemType.isBF16()) {
         if (mfmaVersion == 1)
           kDim = 4;
-        if (mfmaVersion == 2)
+        if (mfmaVersion >= 2)
           kDim = 8;
       }
+      if (elemType.isFloat8E4M3FNUZ() || elemType.isFloat8E5M2FNUZ()) {
```

Comment:
@scxiao hope now I choose correct data types...

fp8 is Float8E4M3FNUZ
bf8 is Float8E5M2FNUZ

is it correct?